autorag/nodes/generator/base.py

```diff
@@ -41,6 +41,9 @@ def wrapper(
 					"You can check valid llm names from autorag.generator_models."
 				)
 			batch = kwargs.pop("batch", 16)
+			if llm == "huggingfacellm":
+				kwargs["model_name"] = kwargs.pop("model")
```

Comment:
What happened the user use both `model` and `model_name`? Then what is become first if two are different.